solver/src/main.rs

```diff
@@ -212,9 +212,13 @@ async fn main() {
     let pool_aggregator = PoolAggregator::from_providers(&pair_providers, &web3).await;
     let pool_fetcher = Arc::new(
         PoolCache::new(
-            args.shared.pool_cache_blocks,
-            0,
-            args.shared.pool_cache_maximum_recent_block_age,
+            PoolCacheConfig {
+                number_of_blocks_to_cache: args.shared.pool_cache_blocks,
+                number_of_pairs_to_auto_update: 0,
```

Comment:
any particular reason we hard code the zero here?

Reply:
I think its because the driver doesn't auto-update pairs on new blocks, but instead forces the cache to get primed with a specific block number on each run. This way all the components during the run dont:
- need to go to the EVM in order to get data
- use consistent data from the same block

Reply:
Maybe good to put this thought into a comment there?

Reply:
> but instead forces the cache to get primed with a specific block number on each run

That is an idea for a future change. Either way the driver does not need auto updating so it is disabled. I already disabled it in the PR that made use of the cache in the driver. Added the comment.